### PR TITLE
Add a recipe for polap v0.2.4

### PR DESCRIPTION
**Pull request by goshng**
Add a recipe for polap v0.2.4

### CODE REVIEW

File: recipes/polap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/goshng/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: "{{ sha256 }}"
+
+build:
+  noarch: python
```
Comment (bgruening) [inline recipes/polap/meta.yaml:14]: Some as in the other PR. You are using R scripts as well, so you? This should be noarch:generic imho

Comment (goshng) [reply recipes/polap/meta.yaml:14]: Thank you. I need to figure out what you mean. It is somewhat steep learning curve up until now. I have basically zero about making a condo package.

Comment (goshng) [reply recipes/polap/meta.yaml:14]: As always, I appreciate your efforts of reviewing the recipe.

Comment (goshng) [reply recipes/polap/meta.yaml:14]: > Some as in the other PR. You are using R scripts as well, so you? This should be noarch:generic imho

You are right about it. I am using R in the script. Let me have time to figure out what the terms are: noarch:generic and imho. Thank you  for your review.

Comment (goshng) [reply recipes/polap/meta.yaml]: I edited the noarch to noarch:generic as recommended. Thank you.

**Review by martin-g — APPROVED** [review]

**Comment by goshng** [comment]
Thank you.

**Pull request merged**